### PR TITLE
Update U&P email templating

### PR DESCRIPTION
**Pull request by Convly**
### What does it do?

- Update regexp used for U&P email templating

### CODE REVIEW

File: packages/plugins/users-permissions/server/services/users-permissions.js [hunk 1]
```diff
@@ -3,6 +3,11 @@
 const _ = require('lodash');
 const { filter, map, pipe, prop } = require('lodash/fp');
```
Comment (innerdvations) [inline packages/plugins/users-permissions/server/services/users-permissions.js:4]: Can we replace lodash with lodash/fp here?

File: packages/plugins/users-permissions/server/controllers/validation/email-template.js [hunk 2]
```diff
@@ -1,8 +1,17 @@
 'use strict';
 
 const _ = require('lodash');
```
Comment (innerdvations) [inline packages/plugins/users-permissions/server/controllers/validation/email-template.js:3]: Can we replace lodash with lodash/fp here?

**Review by innerdvations — DISMISSED** [review]

File: packages/plugins/users-permissions/server/services/users-permissions.js (hunk 1 above)
Comment (Convly) [reply packages/plugins/users-permissions/server/services/users-permissions.js:4]: image
Lodash FP functions are not always interchangeable with regular lodash. In this scenario, we would need to have somehow access to the values' key.

Comment (Convly) [reply packages/plugins/users-permissions/server/services/users-permissions.js:4]: image
One option could be to update those mismatches everywhere with custom flows like this one, but I fear that in the end, we could introduce small bugs or changes that are almost imperceptible during the refactor.

**Review by christiancp100 — APPROVED** [review]
LGTM 🌻

**Review by christiancp100 — APPROVED** [review]

**Pull request merged**